package.json

```diff
@@ -13,6 +13,8 @@
     "ejs": "~2.3.4",
     "finalhandler": "^0.5.0",
     "lodash": "^3.10.1",
+    "moment": "~2.13.0",
```

Comment:
Not sure how crazy I am about adding `moment` as a dependency.  Is there a way to just pass in a string?